price-feeder/oracle/provider/binance.go

```diff
@@ -191,13 +215,27 @@ func (p *BinanceProvider) setTickerPair(ticker BinanceTicker) {
 func (p *BinanceProvider) setCandlePair(candle BinanceCandle) {
 	p.mtx.Lock()
 	defer p.mtx.Unlock()
-	p.candles[candle.Symbol] = candle
+	timePeriod := time.Now().Add(time.Minute*-10).Unix() * 1000
```

Comment:
What is `timePeriod`? We should put this in a private global var.

Reply:
`timePeriod` is the time before which we consider these candles stale - totally agree 👍

Reply:
Did this a bit differently, since putting timePeriod in a global var would make it stale.

Created a new function called `PastUnixTime(minutes int64) (int64)` that returns the unix millisecond time in the past, and a couple constants called `providerCandlePeriod` and `tvwapCandlePeriod`, which the providers and tvwap pass into this function.

f23dfc213d31e56360b36bf38e0fe9425a4a272a